src/albert/resources/projects.py

```diff
@@ -107,6 +108,18 @@ def validate_status(cls, value):
         return value
 
 
+class ProjectSearchItem(BaseModel, HydrationMixin[Project]):
+    id: str | None = Field(None, alias="albertId")
+    description: str = Field(min_length=1, max_length=2000)
+    status: str | None = Field(default=None, exclude=True, frozen=True)
+
+    @field_validator("id", mode="before")
+    def normalize_id(cls, value: str | None) -> str | None:
+        if value and not value.startswith("PRO"):
+            return f"PRO{value}"
+        return value
```

Comment:
You should use the `ProjectId` type annotation to automatically get this validation.